### PR TITLE
blog post on ocrd workspace CLI

### PR DESCRIPTION
**Pull request by kba**
Instead of slides for the input presentation tomorrow.

### CODE REVIEW

File: site/en/_posts/2020-07-15-ocrd-workspace-cli.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+ocrd workspace --directory /tmp/ws add -i FILE_0001_PNG -G OCR-D-IMG -m image/png -g PHYS_0001 /tmp/IMG_0001.png
+```
+
+Here's a breakdown of the options
+
+* `-i FILE_0001_PNG`: This must be a unique identifier of a file. Make sure to make it specific enough, e.g. by adding the format (as we're doing here) or with some other consistent naming scheme
+* `-G OCR-D-IMG`: Add this file to the `OCR-D-IMG` file group (or create it if it does not yet exist). The names of `mets:fileGrp` are **completely arbitrary**. By convention, OCR-D uses `OCR-D-IMG` for the unprocessed images in [the Ground Truth](https://ocr-d.de/gt-repo/) but other conventions call this file group `MAX` (for MAXimum resolution).
+* `-m image/png`: The media type or MIME type of the file, `image/png` for PNG images, `image/tiff` for TIFF, etc.
+* `-g PHYS_0001`: The ID of the physical page, i.e. the `mets:div` that groups together all files representing the same page. The identifier is arbitrary as well, though many METS tools chose `PHYS_` + numerical identifier as the naming convention.
+
```
Comment (EEngl52) [inline site/en/_posts/2020-07-15-ocrd-workspace-cli.md:142]:
```suggestion
Note that -i and -g always have to start with a letter
```

File: site/en/_posts/2020-07-15-ocrd-workspace-cli.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  --page-coordinate-consistency [poly|baseline|both|off]
+                                  How fierce to check PAGE multi-level
+                                  coordinate consistency
+
+  --help                          Show this message and exit.
+```
+
+If you don't provide any options, the validation will run all checks on your
+workspace and report any issues, grouped by severity ("notice", "warning",
+"error") as a simple XML document that is both human-readable bnd
```
Comment (EEngl52) [inline site/en/_posts/2020-07-15-ocrd-workspace-cli.md:196]:
```suggestion
"error") as a simple XML document that is both human-readable and
```

File: site/en/_posts/2020-07-15-ocrd-workspace-cli.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+machine-readable.
+
+To skip certain tests that are not helpful for you, use the `--skip` option.
+For example, if your images have incorrect `DPI` metadata, you can omit those
+messages with `--skip pixel_density`.
+
+## Further reading
+
+* [Setup Guide](https://ocr-d.de/en/setup)
+* [OCR-D Chat](https://gitter.im/OCR-D/Lobby)
```
Comment (EEngl52) [inline site/en/_posts/2020-07-15-ocrd-workspace-cli.md:206]:
```suggestion
* [User Guide](https://ocr-d.de/en/user_guide#preparing-a-workspace)
* [OCR-D Chat](https://gitter.im/OCR-D/Lobby)
```

File: site/en/_posts/2020-07-15-ocrd-workspace-cli.md (hunk 1 above)
Comment (kba) [reply site/en/_posts/2020-07-15-ocrd-workspace-cli.md:142]: ah well spotted, I wanted to add that but forgot.

**Comment by kba** [comment]
https://github.com/OCR-D/ocrd-website/wiki/Intro-ocrd-workspace-CLI

**Pull request closed without merging**